test/test_vanderpol.cpp

```diff
@@ -68,7 +68,7 @@ int VanderPol()
 
     for (;;) {
         r = optsolver.step(modelX, r.cmd);
-        auto seq = optsolver.getOptimalSequence();
+        // auto seq = optsolver.getOptimalSequence();
```

Comment:
`optsolver.getOptimalSequence` should be kept otherwise we are not really testing its implementation in case of NLMPC